src/wmnds/patterns/header-v2/_header-v2.njk

```diff
@@ -79,8 +79,8 @@
                         <svg class="wmnds-mega-menu__sub-menu-link-icon">
                           <use xlink:href="#wmnds-{{subnavItem.icon}}" href="#wmnds-{{subnavItem.icon}}"></use>
                         </svg> 
-                        {%- endif %}
-                        {{subnavItem.name}}</a>
+                        {% endif -%}
+                        {{subnavItem.name | replace("Swift", wmndsIcon({icon: 'swift-full-logo', class: 'wmnds-swift-icon'}) | indent(6) | trim) | safe}}</a>
```

Comment:
I this is having a weird effect in terms of showing the Swift logo twice:

![image](https://user-images.githubusercontent.com/13015201/93211371-0cf8bf80-f759-11ea-9b27-3f70cb1fb286.png)